package.json

```diff
@@ -16,42 +16,42 @@
   },
   "repository": "git@github.com:ciena-frost/ember-frost-pick-list.git",
   "engines": {
-    "node": ">= 5.0.0"
+    "node": ">= 6.0.0"
   },
   "author": "",
   "license": "MIT",
   "devDependencies": {
     "broccoli-asset-rev": "^2.5.0",
+    "chai-jquery": "^2.0.0",
     "ember-ajax": "0.7.1",
     "ember-cli": "~2.8.0",
     "ember-cli-app-version": "^1.0.0",
+    "ember-cli-chai": "0.3.2",
     "ember-cli-dependency-checker": "^1.2.0",
+    "ember-cli-frost-blueprints": "1.2.1",
     "ember-cli-htmlbars-inline-precompile": "^0.3.1",
     "ember-cli-inject-live-reload": "^1.4.0",
-    "ember-cli-jshint": "^1.0.0",
+    "ember-cli-mocha": "^0.10.0",
     "ember-cli-release": "^0.2.9",
     "ember-cli-sri": "^2.0.0",
     "ember-cli-test-loader": "^1.1.0",
     "ember-cli-uglify": "^1.2.0",
+    "ember-concurrency": "0.7.19",
     "ember-data": "~2.8.0",
+    "ember-disable-proxy-controllers": "^1.0.1",
```

Comment:
This can be removed and `"ember-disable-prototype-extensions": "^1.1.0",` should be added.